rotkehlchen/chain/ethereum/modules/makerdao/common.py

```diff
@@ -52,10 +51,11 @@ def _get_account_proxy(self, address: ChecksumEthAddress) -> Optional[ChecksumEt
         reaching it or with the returned result.
         - BlockchainQueryError if an ethereum node is used and the contract call
         queries fail for some reason
+        - DeserializationError
```

Comment:
heh yeah but that's not enough. If it can actually raise this error that means you **have** to handle it somewhere on the call stack.

We can not let errors like that propagate outwards without handling or they will crash the greenlet they run in or the app itself.

By checking the code .. all callers do not check for anything like `DeserializationError`.

So, at least for now, the easiest way to handle this would be to put this in a `try/except` and in the `except` log an error and raise a `RemoteError`. And since you no longer raise `DeserializationError` from here, remove it from the docstring, and adjust the `RemoteError` docstring.

`RemoteError` is handled  properly in the call stack from what I saw.